PlatformConfigurations/POEManagement.md

```diff
@@ -6,3 +6,471 @@ layout: default
 ---
 
 # Power over Ethernet Management
+
+Power over Ethernet (PoE) technology allows network cables to carry electrical power, enabling devices to operate without a separate power supply.
+
+The PoE agent is a software package on DentOS that assists users in managing the PoE chip on their board. This agent ensures that the system can automatically restore user configurations from cold boot, warm boot, or even after a system crash.
+
+This document has three parts: the Installation Guide, the User Guide, which provides guidelines for command operations, and Troubleshooting, which helps handle runtime problems.
+
+## Installation
+
+To install PoE agent, follow these steps:
+
+### 1. Update the package list:
+
+```
+sudo apt-get update
+```
+
+### 2. Install necessary packages:
+
+```
+sudo apt-get install python3-stdeb fakeroot python-all dh-python
+```
+
+### 3. Clone the PoE project repository:
+
+```
+git clone https://github.com/dentproject/poed.git
+cd poed
+```
+
+### 4. Build the Debian package:
+
+```
+bash build-deb.sh
+```
```

Comment:
After this, you should show how to start the POE agent since the majority of the instructions below (including help) needs the agent running. 
    systemctl restart poed

It may also help if we refer them to the troubleshoot section at the end of the doc in case the agent does not start.

Reply:
https://github.com/dentproject/docs.dent.dev/pull/54/commits/e3d9b8e6c1b416321cc749c882f562f4ea0c889b